build.gradle.kts

```diff
@@ -42,9 +42,9 @@ fun getDate(): String {
     return format.format(Date())
 }
 
-version = "$mcVersion-$modVersion-$buildNumber"
+version = "$modVersion-$buildNumber"
```

Comment:
I don't see minecraft version in `archivesBaseName`. Is it added to filename anyway?

Reply:
Oops, missed that one. Will fix!